src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -1758,6 +1776,10 @@
 examples:
   - name: Create an empty MX record set.
     text: az network dns record-set mx create -g MyResourceGroup -z www.mysite.com -n MyRecordSet
+  - name: Create an empty MX record set. (autogenerated)
+    text: |
+        az network dns record-set mx create --name MyRecordSet --resource-group MyResourceGroup --ttl 30 --zone-name www.mysite.com
+    crafted: true
```

Comment:
The commands in the autogenerated document has a few more parameters. Could the name describe these parameters more, so as to be different from the above name.